i2c/oled_i2c/README.adoc

```diff
@@ -1,36 +1,36 @@
+The SSD1306 is not operated via direct reads/writes to registers. Instead, a list of versatile commands is defined (see datasheet) that allows the user to access all the capabilities of the driver. Ultimately, these commands end up writing to internal registers but the key point is that the I2C write process is now different. After sending a slave address, the data that follows can be either a command, flags to follow up a command or data to be written directly into the display's RAM. A control byte is required for each write after the slave address so that the driver knows what type of data is being sent.
 
-This display is 32 pixels high by 128 pixels wide. These 32 pixels are partitioned into 4 pages, each 8 bits in height. In RAM, This looks roughly like:
+This display is 32 pixels high by 128 pixels wide. These 32 vertical pixels are partitioned into 4 pages, each 8 pixels in height. In RAM, This looks roughly like:
 
 ----
-           | COL0 | COL1 | COL2 | COL3 | ...
-    PAGE 0 |      |      |      |      |
-    PAGE 1 |      |      |      |      |
-    PAGE 2 |      |      |      |      |
-    PAGE 3 |      |      |      |      |
-    ------------------------------------
+           | COL0 | COL1 | COL2 | COL3 |  ...  | COL126 | COL127 |
+    PAGE 0 |      |      |      |      |       |        |        |
+    PAGE 1 |      |      |      |      |       |        |        |
+    PAGE 2 |      |      |      |      |       |        |        |
+    PAGE 3 |      |      |      |      |       |        |        |
+    --------------------------------------------------------------
 ----
 
 Within each page, we have:
 
 ----
-          | COL0 | COL1 | COL2 | COL3 | ...
-    COM 0 |      |      |      |      |
-    COM 1 |      |      |      |      |
-       :  |      |      |      |      |
-    COM 7 |      |      |      |      |
-    ------------------------------------
+          | COL0 | COL1 | COL2 | COL3 |  ...  | COL126 | COL127 |
+    COM 0 |      |      |      |      |       |        |        |   
+    COM 1 |      |      |      |      |       |        |        |   
+       :  |      |      |      |      |       |        |        |  
+    COM 7 |      |      |      |      |       |        |        |
+    -------------------------------------------------------------
 ----
 
-Note the difference between columns in RAM and the actual segment pins on the display. The distinction between these two is important as we can remap, or add offsets between the two in the driver itself instead of moving pixels in our frame buffer code.
+Note the difference between columns in RAM and the actual segment pads that connect the driver to the display. The distinction between these two is important as we can remap, or add offsets between the two in the driver itself instead of moving pixels in our frame buffer code.
```

Comment:
I'm still not sure what this paragraph is trying to say? :confused:   (maybe that's because I've not read the SSD1306 datasheet?)

Reply:
The display's pins are physically connected to the driver via segment and common pins. The RAM addresses COL0 - COL127 are then mapped to these segment pins SEG0 - SEG127. But we can change the mapping, to easily mirror the image, etc without having to write a flipped buffer to the RAM again.

Reply:
Perhaps this is another "behind the scenes" detail that the user doesn't need to know? WDYT @aallan ?

Reply:
Sounds like good detail to have in there, but it's definitely an aside. Should be in a `[NOTE]` block.